### PR TITLE
Add new events_exporter version and events logging

### LINKED ISSUE #2022: [extended-monitoring] Add logging events to events exporter
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Use case. Why is this important?

In many situations, event messages are one of the main parts of debugging. But if the event-exporter exposes messages to prometheus tags, the cardinality will increase several times. Thus, logging will be the easiest way to export event messages.

### Proposed Solution

[This PR](https://github.com/nabokihms/events_exporter/pull/1) implements the log flag for the event-exporter. In Deckhouse cm, I suggest adding an ```extendedMonitoring.events.logEvents``` bool flag to control log exposing

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by alex123012**
## Description

New version of events_exporter supports logging events in debug log-level
## Why do we need it, and what problem does it solve?

In many situations, event messages are one of the main parts of debugging. But if the event-exporter exposes messages to prometheus labels, the cardinality will increase several times. Thus, logging will be the easiest way to export event messages. 
## What is the expected result?
Logging events from events_exporter pod to stdout

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: extended-monitoring
type: feature
summary: Add events logging to stdout in events_exporter.
impact_level: default
```

close #2022

### CODE REVIEW

File: modules/340-extended-monitoring/docs/README_RU.md [hunk 1]
```diff
@@ -6,5 +6,5 @@ title: "Модуль extended-monitoring"
 
 - `extended-monitoring-exporter` — генерирует метрики и [алерты](configuration.html#non-namespaced-kubernetes-objects) по свободному месту и inode на нодах, плюс включает «расширенный мониторинг» объектов в указанных Namespace.
 - `image-availability-exporter` — генерирует метрики о проблемах доступа к образу в container registry.
-- `events-exporter` — собирает события в кластере Kubernetes и отдает их в виде метрик.
+- `events-exporter` — собирает события в кластере Kubernetes и отдает их в виде метрик. Также логирует все эвенты.
```
Comment (nabokihms) [inline modules/340-extended-monitoring/docs/README_RU.md:9]:
```suggestion
- `events-exporter` — собирает события в кластере Kubernetes и отдает их в виде метрикю
```
```suggestion
- `events-exporter` — собирает события в кластере Kubernetes и отдает их в виде метрик.
```

File: modules/340-extended-monitoring/docs/README.md [hunk 2]
```diff
@@ -6,5 +6,5 @@ Contains the following Prometheus exporters:
 
 - `extended-monitoring-exporter` — generates free space / inode-related metrics and [alerts](configuration.html#non-namespaced-kubernetes-objects); also, it enables the "extended monitoring" of objects in the selected Namespaces.
 - `image-availability-exporter` — generates metrics about issues with accessing images in the container registry.
-- `events-exporter` — collects Kubernetes cluster events end exposes them as metrics.
+- `events-exporter` — collects Kubernetes cluster events end exposes them as metrics. Also logs all events.
```
Comment (nabokihms) [inline modules/340-extended-monitoring/docs/README.md:9]:
```suggestion
- `events-exporter` — collects Kubernetes cluster events end exposes them as metrics.
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @nabokihms. I've started the workflow for label 'e2e/run/gcp'...

**Comment by nabokihms** [comment]
@alex123012 thanks, well done!

**Pull request merged**